covid19/tasks.py

```diff
@@ -10,6 +11,11 @@ def process_new_spreadsheet_task(spreadsheet_pk):
     except StateSpreadsheet.DoesNotExist:
         return None
 
-    print(f"Processing {spreadsheet}...")
+    try:
+        spreadsheet.warnings = validate_historical_data(spreadsheet)
```

Comment:
É muito importante que as validações históricas também fiquem no form, assim o usuário só "sairá" do form se conseguir fazer upload sem erros ou se desistir, ou seja, podemos contarr sempre que o registro que for adicionado em `Spreadsheet` estará com as devidas checagens automáticas já feitas (isso simplifica o processo do fluxo das checagens e notificações, além de ser mais user-friendly).

O que vai ficar pra essa task é a parte de:
1- Se nenhum outro usuário subiu alguma planilha pra esse estado/data, notifica no chat aos responsáveis que precisamos de uma planilha pra checar;
2- Se sim, verifica se ambas as planilhas são iguais:
2.1- Se não forem, notifica no chat que precisamos que alguém suba uma pra desempate;
2.2- Se forem, inicia o processo de deployment desse estado.

Reply:
Resolvido em:

- 49bab8d 
- abd1b1a 
- 5c635c2 

Além disso, adicionei a listage de warnings no admin
![Screenshot from 2020-04-16 16-06-18](https://user-images.githubusercontent.com/238223/79496597-bb5ad600-7ffc-11ea-9f82-b6a49996f83b.png)
 conforme o print abaixo: